.travis.yml

```diff
@@ -27,10 +33,15 @@ matrix:
 before_install:
   - yarn global add bower
   - bower --version
-  - yarn add phantomjs-prebuilt
-  - node_modules/phantomjs-prebuilt/bin/phantomjs --version
+  - export PATH=/usr/local/phantomjs-2.0.0/bin:$PATH
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
 
 install:
+  - mkdir travis-phantomjs
```

Comment:
It seems in the testem config you updated it to not use phantomjs at all — which is 👍 , but I think this needs to be updated and cleaned up as well. You can probably just use Travis's normal method for making sure Chrome is available: https://docs.travis-ci.com/user/chrome

Reply:
Okay cool, looks like this was left over as I updated the Travis config in stages. I'll clean up the `.travis.yml` file.